flang/test/Semantics/OpenMP/combined-constructs.f90

```diff
@@ -33,7 +33,7 @@ program main
   enddo
   !$omp end target parallel
 
-  !ERROR: The DEFAULTMAP clause requires a variable-category SCALAR in OpenMP v1.1, try -fopenmp-version=50
+  !ERROR: A variable-category modifier is required
```

Comment:
I'm worried that the new error messages are less informative than the old ones

Reply:
They can be improved.  Right now `OmpVerifyModifiers` checks all properties, and returns a single value: "everything passed" or "something is wrong".  Based on the descriptor information, it could generate the "try different version" if applicable, plus it could return a more detailed information about what failed to the caller.

On the positive side, with the new functions, the actual modifier is highlighted (except if it's required but missing).

If you're ok with that, I can improve this in a subsequent PR.

Reply:
Improving in a subsequent PR is okay with me.